lib/charms/opensearch/v0/opensearch_users.py

```diff
@@ -22,18 +24,29 @@ class OpenSearchUserMgmtError(Exception):
     """Base exception class for OpenSearch user management errors."""
 
 
-class OpenSearchUserManager:
+class OpenSearchUserManager(Object):
```

Comment:
Why are we extending `Object` here?

Reply:
I think you already have the charm, so probably `self.model = self.charm.model` and removing the `Object`?